product-mini/platforms/nuttx/wamr.mk

```diff
@@ -187,9 +187,7 @@ CSRCS += utils.c
 VPATH += $(IWASM_ROOT)/libraries/debug-engine
 endif
 
-ifeq ($(CONFIG_INTERPRETERS_WAMR_STACK_GUARD_SIZE),)
-CFLAGS += -DWASM_STACK_GUARD_SIZE=0
```

Comment:
what's the default value of DWASM_STACK_GUARD_SIZE?

Reply:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/5a75c31c38d3e799ae12165dec38454674d8c81e/core/config.h#L418